model/host/host_test.go

```diff
@@ -1250,9 +1251,80 @@ func TestFindAllRunningParents(t *testing.T) {
 	assert.NoError(host7.Insert())
 	assert.NoError(host8.Insert())
 
-	containers, err := FindAllRunningParents()
+	hosts, err := FindAllRunningParents()
+	assert.NoError(err)
+	assert.Equal(3, len(hosts))
+
+}
+
+func TestFindAllRunningParentsOrdered(t *testing.T) {
+	assert := assert.New(t)
+	assert.NoError(db.ClearCollections(Collection))
+
+	const d1 = "distro1"
+	const d2 = "distro2"
+
+	host1 := &Host{
+		Id:                      "host1",
+		Distro:                  distro.Distro{Id: d1},
+		Status:                  evergreen.HostRunning,
+		HasContainers:           true,
+		LastContainerFinishTime: time.Now().Add(10 * time.Minute),
+	}
+	host2 := &Host{
+		Id:                      "host2",
+		Distro:                  distro.Distro{Id: d1},
+		Status:                  evergreen.HostStarting,
+		LastContainerFinishTime: time.Now().Add(10 * time.Minute),
```

Comment:
Should a host that is not a parent have a `LastContainerFinishTime` field?

Reply:
Good catch! Test behaves as expected but I will delete that line for clarity.